docs/getting_started.md

```diff
@@ -68,6 +68,21 @@ The API server can then be accessed using the localhost:8080
 Depending on the Argo CD version you are installing, the method how to get the
 initial password for the `admin` user is different.
 
+> Argo CD v1.9 and later
```

Comment:
I think with the versioned docs in place, we can get rid of the "if v > x else" blocks for good :)